### ISSUE TITLE
Won't work

### ISSUE THREAD

**Issue by Syng22** [shmykelsa/AAAD#153]
I have a s21 ultra,  2019 camaro ss. None of the screen mirroring will work now. It'll just pop up that the app isn't working

**Comment by realrodri** [comment]
Mine too. My S21 Ultra won't open any app. It just shows Fermata Auto, and no more apps are shown. When I try to open Fermata Auto, it just crashes. It doesn't even open.

**Comment by ericalarcon0512** [comment]
I also have the galaxy s21 ultra and everything downloaded perfectly but when I press the app on the radio it dont open and the phone says app keeps stopping pls help I even tried downloading other app n they all do the same thing

**Comment by realrodri** [comment]
I refunded it. I'm using my iPhone XR with CarBridge (jailbroken) to watch films and TV shows. It's a shame it does not work with this device.

**Comment by ericalarcon0512** [comment]
Did they get back to you with a fix or anything? @realrodri

**Comment by realrodri** [comment]
> Did they get back to you with a fix or anything? @realrodri 

Not at all. "ReAd tHe FaQ". PayPal returned my money and I uninstalled all the apps and AAAD as I stated in the dispute.
